lib/src/resource_owner_password_grant.dart

```diff
@@ -56,13 +61,13 @@ Future<Client> resourceOwnerPasswordGrant(
     }
   }
 
-  if (scopes != null && !scopes.isEmpty) body['scope'] = scopes.join(' ');
+  if (scopes != null && !scopes.isEmpty) body['scope'] = scopes.join(delimiter ?? ' ');
```

Comment:
Okay, just pushed commits to correct these. Let me know if they're up to your expectations!